### DISCUSSION TITLE
pg_stream: support for partitions (TimescaleDB)

### DISCUSSION THREAD

**Discussion by ntx-ben** [redpanda-data/connect#3045] [Ideas]
Are there any plans to eventually support CDC of partitions (more specifically TimescaleDB Hypertables). Currently, the `pg_stream` connector is super useful to stream data from regular Postgres tables, however it unfortunately is incompatible with partitions. I found another Golang project that does CDC of Hypertables however it would be nice if this was built into connect's `pg_stream` input connector.

I understand this is a big ask but I'm just wondering.

Thanks,

CC @rockwotj @le-vlad

**Comment by le-vlad** [comment]
Hey, @ntx-ben 
Can you share the project you found? I'd like to take a look.

**Reply by ntx-ben** [reply]
Here: https://github.com/noctarius/timescaledb-event-streamer
